rbac/management/group/view.py

```diff
@@ -256,6 +256,7 @@ def create(self, request, *args, **kwargs):
             }
         """
         validate_group_name(request.data.get("name"))
+        check_duplicate_entry(request.data.get("name"), self.request.tenant)
```

Comment:
@EvanCasey13 is it possible to consider this [comment](https://github.com/RedHatInsights/insights-rbac/pull/1217#discussion_r1786771247)  ?

it looks that there are some benefits with that.

Reply:
Yes I think I can catch IntegrityException when create occurs will go that route and remove the function if it works